CMakeLists.txt

```diff
@@ -94,6 +94,10 @@ endif()
 set(CUTLASS_ENABLE_SYCL OFF CACHE BOOL "Enable SYCL")
 set(CUTLASS_SYCL_PROFILING_ENABLED OFF CACHE BOOL "Use SYCL events to calculate device execution time")
 
+set(SWITCH_WG OFF CACHE BOOL "Enable SWITCH WG")
```

Comment:
Use `CUTLASS_SYCL_SWITCH_WG` instead. Also mention in the description that this is a flag means to experiment with different configurations for GEMM on Intel PVC during benchmarking.